exonum-dictionary.txt

```diff
@@ -130,5 +130,6 @@ uuids
 validator
 validator's
 validators
+veyor
```

Comment:
I propose to rename the badge title to "App**v**eyor Build Status" (it isn't visible anyway), so we wouldn't need to add such strange "word" to the exceptions.